website/docs/reference/resource-configs/snowflake-configs.md

```diff
@@ -9,6 +9,373 @@ To-do:
 - use the reference doc structure for this article / split into separate articles
 --->
 
+## Iceberg table format <Lifecycle status="beta"/>
+
+Snowflake Iceberg tables support for the dbt-snowflake adapter is available for three out-of-the-box materializations:  
+- [Table](/docs/build/materializations#table)
+- [Incremental](/docs/build/materializations#view)
+- [Dynamic](#dynamic-tables) 
+
+For now, Iceberg tables require a [behavior flag](/reference/global-configs/behavior-changes) due to performance impact related to using Iceberg tables. Snowflake does not support `is_iceberg` on the  `Show Objects` queries, which dbt depends on for metadata.
+
+To use Iceberg, set the Iceberg Materialization Behavior flag to `True` in your dbt_project.yml:
```

Comment:
is "Iceberg Materialization Behavior flag" going to be the file name for this behavior flag? only asking bc it doesn't match that actual flag `enable_iceberg_materializations`.

Reply:
Not sure yet. @amychen1776 Thoughts?

Reply:
Two thoughts:
1. @matthewshaver we need a page for this specific behavior flag under Snowflake (related to your other PR)
2. let's update this to say set the `enable_iceberg_materializations` behavior flag to 'True'